markymark/Classes/Layout Builders/UIView/Views/ContainerView.swift

```diff
@@ -31,7 +31,13 @@ class ContainerView: UIView {
 
         constraints += NSLayoutConstraint.constraints(withVisualFormat: "H:|-(left)-[view]-(right)-|", options: [], metrics: metrics, views: views)
         constraints += NSLayoutConstraint.constraints(withVisualFormat: "V:|-(top)-[view]-(bottom)-|", options: [], metrics: metrics, views: views)
-
+        if minHeight != nil {
```

Comment:
use if let minimumHeight = minimumHeight {

}

instead of if != nil and force unwrapping